l3build.1

```diff
@@ -77,6 +77,8 @@ Shows the full log of the failure with 'halt-on-error'
 Shows the invocation to update failing .tlg files
 .IP --shuffle
 Shuffles order of tests
+.IP --stdengine|-s
+Run tests only with the standard engine
```

Comment:
I think it wouldn't hurt if one mentions in places like this that this can differ from config file to config file in a complex setup